src/battle_main.c

```diff
@@ -4191,6 +4191,7 @@ static void HandleTurnActionSelectionState(void)
     for (battler = 0; battler < gBattlersCount; battler++)
     {
         u32 position = GetBattlerPosition(battler);
+        u32 isAiRisky = AI_THINKING_STRUCT->aiFlags[battler] & AI_FLAG_RISKY; // Risky AI switches aggressively even mid battle
```

Comment:
Since we can't do it at function scope it makes more since to do it inside `STATE_TURN_START_RECORD` because otherwise it will assign the variable for each state